compiler-plugin/src/main/kotlin/arrow/meta/quotes/ClassOrObject.kt

```diff
@@ -13,6 +13,55 @@ import org.jetbrains.kotlin.psi.psiUtil.getValueParameters
 import org.jetbrains.kotlin.psi.psiUtil.modalityModifierType
 import org.jetbrains.kotlin.psi.psiUtil.visibilityModifierType
 
+/**
+ * An extension function of [Meta] and inheriting from [ExtensionPhase], [classOrObject] was designed to to feed in
+ * virtually any kind of [KtClass] predicate, followed by mapping a function that takes the desired [Scope] of our
+ * [KtClass] to change whatever PSI elements desired.
+ *
+ * @param [match] designed to to feed in any kind of [KtClass] predicate returning a [Boolean]
```

Comment:
Can't seem to figure out why the param keeps shoving down the description to the next line!

Reply:
`@param`, `@return` etc should be at the of the KDoc as convention, no?

Not sure if this is related to "shoving down the description", I am not sure I understand what you mean  by that :D

Reply:
Ah yes, it's happening definitely in the screenshot, regardless of whether I'm doing `@param match ...` or `@param [match] ....`

It should be part of the KDocs yeah!

Reply:
I'm sorry I meant:
Shouldn't be at the end/bottom of the KDoc.
[Example in Arrow Fx Concurrent](https://github.com/arrow-kt/arrow/blob/c3fa9ec8567da5bfbd9ecaac2bc055c908dfac00/modules/fx/arrow-fx/src/main/kotlin/arrow/fx/typeclasses/Concurrent.kt#L138)